engine/src/main/java/org/camunda/bpm/engine/impl/batch/AbstractBatchJobHandler.java

```diff
@@ -41,19 +42,80 @@
 
   @Override
   public boolean createJobs(BatchEntity batch) {
-    CommandContext commandContext = Context.getCommandContext();
-    ByteArrayManager byteArrayManager = commandContext.getByteArrayManager();
-    JobManager jobManager = commandContext.getJobManager();
-
     T configuration = readConfiguration(batch.getConfigurationBytes());
+    return doCreateJobs(batch, configuration);
+  }
+
+  protected boolean doCreateJobs(BatchEntity batch, T configuration) {
```

Comment:
What is the purpose of having this method separated?

Reply:
Thanks for the hint, I used that method a long time for backwards compatibility implementations but with a refactoring this was not necessary anymore. I missed to reintegrate that method. I will reintegrate the code and remove the extra method. 👍